GuildedRoseLLC/Items/StaticItemRepository.swift

```diff
@@ -4,20 +4,8 @@ public class StaticItemRepository : ItemRepository {
     public init() {}
     
     public func getItems(onSuccess: @escaping (_:[Item]) -> Void){
-        var items: [Item] = []
+        let items: [Item] = Item.testData
```

Comment:
Is `Item.testData` even needed anymore? Could we just hardcode the items in this class itself?

Having the `Item` class be responsible for the `testData` seems like a violation of SRP.